src/lib/types/models/api-token.ts

```diff
@@ -37,6 +37,10 @@ export interface IApiToken extends Omit<IApiTokenCreate, 'alias'> {
     alias: string | null;
 }
 
+export interface IApiTokenExt extends IApiToken {
```

Comment:
I struggle a bit to understand the different use cases for `IApiToken` vs `IApiTokenExt`. Can you explain how they are different and when would we use `IApiTokenExt` without a `tokenName` when we will always map username to the tokenName?

I am left wondering if we in our internal model should just rename `username` to `tokenName` and just accept either one on creation, but keep it as `tokenName` internally.

Reply:
I wanted to get some discussion going around this, but at the same time have something reasonably functional that had a minimal effect on the type system close to the db. (in case there were some reflection-schema magic that could trip us up). So I kept the changes as external and unobtrusive as possible for the initial conversation.

> I am left wondering if we in our internal model should just rename username to tokenName and just accept either one on creation, but keep it as tokenName internally.
Should we also rename the database field if so?